javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------*/
+// Age - ify(A future age calculator)
+const yearOfBirth = 1987;
+const yearFuture = 2045;
+
+const age = yearFuture - yearOfBirth;
+
+console.log(`You will be ${age} years old in ${yearFuture}`);
+
+
+/*---------------------------------------------------------------------------------------------*/
+// Goodboy - Oldboy(Adog age calculator)
+const dogYearOfBirth = 2021;
+const dogYearFuture = 2045;
+
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears == true) {
+    console.log(`\nYour dog will be ${dogYear}  human years old in ${dogYearFuture}`);
+}
+
+/*---------------------------------------------------------------------------------------------*/
+// Housey pricey (A house price estimator)
+var petersHouseCurrentPay = 2500000;
+var juliasHouseCurrentPay = 1000000;
+
+
+function volumnInMeters(w, d, h) {
+    return (w * d * h) * 2.5 * 1000;
+}
+
+function gardenSizeInM2(garden) {
+    return garden * 300;
+}
+
+console.log(`\nJulia's current rent is: ${juliasHouseCurrentPay}`);
+
+let juliesRightPayment = volumnInMeters(5, 11, 8) + gardenSizeInM2(70);
+console.log("Julia's right amount of rent is: " + `${juliesRightPayment}`);
+
+if (juliesRightPayment > juliasHouseCurrentPay) {
+    console.log("It is good amount to pay compare your place's size Juila!");
+} else {
+    console.log("JULIA, YOU NEED TO RE-CONSIDER YOUR PLACE");
+}
+
+console.log(`\nPeter 's current rent is: ${petersHouseCurrentPay}`);
+
+var petersRightPayment = volumnInMeters(8, 10, 10) + gardenSizeInM2(100);
```

Comment:
Is better to avoid "var" declarations because it can be globally or function scooped and can be re-declared. Let's say "var" is too 'flexible', we want something more constant 😅.

Instead of "var" is better to stick with "const" and if you need something a bit momore flexible you can use "let"

There is nice article about it 
https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Reply:
Thanks for the review =)
Changed all 'var' to 'let' or 'const'.
Please check my revision